documentation/source/tutorials/pages/registering-different-data-from-the-same-session.rst

```diff
@@ -0,0 +1,189 @@
+   :align: center
+   :figwidth: 65%
+
+   ``Template->T2`` (blue) + ``T2->MT1`` (green) = ``Template->MT1`` (the desired result)
+
+For the purpose of this tutorial, we will treat the example MT data and the example T2 data as though they were acquired in the same session. So, we will be able to reuse the ``/t2/warp_template2anat.nii.gz`` warping field generated in :ref:`registration-to-template`.
+
+To accomplish this, we now use the ``sct_register_multimodal`` command, which is designed to co-register two images together.
+
+.. code:: sh
+
+   sct_register_multimodal -i $SCT_DIR/data/PAM50/template/PAM50_t2.nii.gz -iseg $SCT_DIR/data/PAM50/template/PAM50_cord.nii.gz \
+                           -d mt1.nii.gz -dseg mt1_seg.nii.gz \
+                           -m mask_mt1.nii.gz -initwarp ../t2/warp_template2anat.nii.gz \
+                           -param step=1,type=seg,algo=centermass:step=2,type=seg,algo=bsplinesyn,slicewise=1,iter=3  \
+                           -owarp warp_template2mt.nii.gz -qc ~/qc_singleSubj
+
+   # Input arguments:
+   #   - i: Source image. Here, it is the PAM50 template taken from the SCT installation directory. The T2 version
+   #        of the template is used due to its similarity in contrast to the MT1 data.
+   #   - iseg: Segmentation corresponding to the source image. Here, it is the segmented spinal cord volume from
+   #           the PAM50 template, taken from the SCT installation directory.
+   #   - d: Destination image.
+   #   - dseg: Segmentation corresponding to the destination image.
+   #   - m: Mask image, which is used on the destination image to improve the accuracy over the region of interest.
+   #   - initwarp: Initial warping field to apply to the source image. Here, we supply the 'warp_template2anat.nii.gz'
+   #               file that was generated in the previous tutorial. Because we begin with the 'Template->T2'
+   #               transform already applied, the warping field that is generated here will be 'Template->T2->MT1'
+   #               a.k.a. 'Template->MT1'.
+   #   - param: Here, we will tweak the default registration parameters to specify a different nonrigid deformation.
+   #            The important change is 'algo=centermass': Because the template object is already "preregistered"
+   #            from the previous tutorial (see '-initwarp'), the benefits of the default 'algo=centermassrot' have
+   #            already been applied. So, we specify 'algo=centermass' in step 1 to exclude the unnecessary rotation.
+   #   - owarp: The name of the output warping field. This is optional. If not supplied, the filename would be
+   #            generated from the filenames '-i' and '-d', which in this case would be 'warp_PAM50_t22mt1.nii.gz'.
+   #   - qc: Directory for Quality Control reporting. QC reports allow us to evaluate the results slice-by-slice.
+
+   # Output files/folders:
+   #   - mt1_reg.nii.gz: TODO: Empty file. How to explain?
```

Comment:
Interestingly, I noticed that `sct_register_multimodal` outputs an empty file. I think this is because `initwarp` was specified, so the registration was only done one-way. But, if we're not coregistering, then we probably shouldn't output 2 `_reg.nii.gz` files! This might be a bug.